Servers/patching-as-a-service.md

```diff
@@ -1,281 +1,296 @@
+
+### Support
+
+If you have any concerns with the use or affects of this product, please submit a support ticket to CenturyLink Cloud.
+
+If you have suggestions for improvement, please submit a feature request.
+
+### Pricing
+
+This service is free of charge.
+
+### Execution Options
+
+There are three ways to deploy the package, described below. With a Blueprint, the script package can be executed against one VM at a time. Control Portal's server group action functionality can execute against all VMs in a server group. An API request can deploy against all VMs a user is authorized to administer under a single alias.
+
+For any method you choose, the following table will assist you with data needed for execution.
+
+
+**Operating Systems** | **Blueprint Name** | **Script Package Name** | **Package ID**
+ --- | --- | --- | ---
+ Windows 2012 and 2012R2 | Auto Patching Windows 2012 | Auto Patching Windows 2012 | b229535c-a313-4a31-baf8-6aa71ff4b9ed
+ Red Hat Enterprise Linux 5, 6, and 7 OR CentOS 5 and 6 | Yum Update Script | Yum Update | 5d743f04-a9ce-4174-a7c8-52df93c47c08
+
+
+### Option 1: Blueprint
+
+Using a Blueprint, you can deploy patches against a single server through Control Portal.
+
+#### Steps
+
+1\. Maintenance Mode
+
+The process initiated by the script package may include several, automated reboots, so - regardless how you choose to execute the package - please disable alerts by putting the server(s) in maintenance mode first.
+
+2\. Select the Blueprint
+
+Locate and select the appropriate Blueprint within the Blueprint Library. See the table above for the name of the Blueprint.
+
+![Windows Update Scripts Blueprint Image](../images/Patching/PatchaaS_WindowsUpdateHoverProd.png)
+
+![YUM Update Script Blueprint Image](../images/Patching/PatchaaS_YumUpdateBP.png)
+
+
+
+3\. Click the Deploy Blueprint Button
+
+4\. Set required parameters
+
+Select whether you want to run under the default server administrator or specify credentials. Select the server you want to patch from the drop down. Press the green “next step 2” button.
+
+5\. Review and Confirm the Blueprint
+
+6\. Deploy the Blueprint
+
+You will see the deployment details stating the Blueprint is queued for execution. This will kick off the Blueprint deploy process and load a page where you can track the deployment progress. Deployment times may vary. Please wait for the build queue to update deployment status.
+
+7\. Deployment Complete
+
+The execution will continue even after the completion of the script so please leave your server in maintenance mode until you receive an email informing you of completion.
+
+8\. Execution Complete
+
+After the patching is complete you will receive an email that patching is complete. Please remove the server from maintenance mode.
+
+9\. Optional: Remove DNS
+
+You may wish to remove DNS 8.8.8.8 from the server.\*

```

Comment:
is this really highlighted enough?  the complaints I've gotten from customers and SE's is that we are not up front that this change even occurs.  i'd think we should highlight anything that modifies the VM at the top of the KB in a category so no one is surprised this even occurs. particularly as this can cause adverse private DNS issues.  burying it in a optional item doesn't really do much.  that's my view